runtime/polkadot/src/lib.rs

```diff
@@ -394,20 +399,38 @@ impl democracy::Trait for Runtime {
 	type VotingPeriod = VotingPeriod;
 	type MinimumDeposit = MinimumDeposit;
 	/// A straight majority of the council can decide what their next motion is.
-	type ExternalOrigin = collective::EnsureProportionAtLeast<_1, _2, AccountId, CouncilCollective>;
+	type ExternalOrigin = system::EnsureOneOf<AccountId,
```

Comment:
Dumb question, but would this `AccountId` allow any account to take on this origin? The following two lines make sense to me.

Reply:
no - it's an overhang from rust's less-than-ideal type inference; we need to provide it explicitly, but it's not one of the origins.